### PR TITLE
prepare v2.0.0 release

### PR DESCRIPTION
**Pull request by FredKSchott**
## v2.0.0 [10-16-2017]
 - [BREAKING] `Workspace.init()` options have been moved to the constructor.
 - [BREAKING] `Workspace` dependency installation must be called manually via
 `Workspace.installBowerDependencies()`.
 - [BREAKING] `Workspace` Class is now responsible for setup & initialization only. All
 post-setup operations (`pushChangesToGithub`, `commitChanges`, etc.) have been
 pulled off as seperate operations.
 - `run()`: New batch-running support for running commands across multiple repos.
 - Updated README.

### CODE REVIEW

**Comment by usergenic** [comment]
Except "v2.0.0 [10-16-2017] <-- 12/18/2017"

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
 
+## v2.0.0 [10-16-2017]
+- [BREAKING] `Workspace.init()` options have been moved to the constructor.
+- [BREAKING] `Workspace` dependency installation must be called manually via
+`Workspace.installBowerDependencies()`.
+- [BREAKING] `Workspace` Class is now responsible for setup & initialization only. All
+post-setup operations (`pushChangesToGithub`, `commitChanges`, etc.) have been
+pulled off as seperate operations.
+- `run()`: New batch-running support for running commands across multiple repos.
+- Updated README.
```
Comment (rictic) [inline CHANGELOG.md]: Recommend adding a .github/PULL_REQUEST_TEMPLATE with a checkbox to check for changelog updating.

This is the only way I ever remember to update any changelogs -.-

Comment (usergenic) [reply CHANGELOG.md]: +1

Comment (FredKSchott) [reply CHANGELOG.md]: +1000, grabbing from tools-common now

**Pull request merged**